### ISSUE TITLE
Window build exe opening the chrome window

### ISSUE THREAD

**Issue by brpradeepprabhu90** [guark/guark#53]
Hi,

Thanks for creating this tool. Love to work. 

After I build the windows EXE when I open it for the first time it is working fine. Without closing the window when I open second-time multiple windows appear and only one window contains actual data the other screens are blank.

After I close all the window, every time i click on EXE it is opening in chrome instead of an inbuilt window and it is showing "Chrome is controlled by automated software"

![image](https://user-images.githubusercontent.com/55737883/123180166-be4f2300-d458-11eb-88d7-199b4563b2ce.png)

**Comment by melbahja** [comment]
when you close the app, is windows kills it's process or stay working in the background?

**Comment by brpradeepprabhu90** [comment]
Yes, It seems like that.  Do i want to check any specific process in background

**Comment by melbahja** [comment]
can you provide your chrome version and after closing the the app can you confirm that chrome not working in the background

**Comment by brpradeepprabhu90** [comment]
I have updated the chrome and restarted the machine

![image](https://user-images.githubusercontent.com/55737883/123346040-629e9b80-d525-11eb-9e5d-1c6e762c479d.png)

The result is the same

**Comment by melbahja** [comment]
I mean in the process manager when you close the app is the chrome process stays in the background or not.

also can you try to change the engine in from chrome to `webview`

**Comment by pavel-zhdanov** [comment]
Hi. got the same problem. there were no chrome processes before starting the application

**Comment by pavel-zhdanov** [comment]
if you change the id in the guark.yaml, then the first launch does not create an extra window
